content/guides/development/rails/index.mdx

```diff
@@ -23,6 +23,18 @@ require "view_component"
 require "primer/view_components/engine"
 ```
 
+It's important to load css directly from the gem. Otherwise lots of components will have a "broken css" look.
+
+Load it as js module into the pipeline. Update app/assets/config/manifest.js with:
+```js
+//= link primer_view_components.css
+```
+
+so you can use this inside the template
+```erb
+<%= stylesheet_link_tag("primer_view_components") %>
+```
+
```

Comment:
Thanks for this addition! I'm suggesting a slight rewording:


```suggestion
**CSS:**

Load CSS as a JS module into the pipeline. Update app/assets/config/manifest.js with:

```js
//= link primer_view_components.css
```

Import the stylesheet into templates:

```erb
<%= stylesheet_link_tag("primer_view_components") %>
```

```

Reply:
@langermank thank you for your comments.
I have applied your suggestions and added a bunch of small things, so now the doc looks like this
<img width="937" alt="Screenshot 2023-09-12 at 11 20 12" src="https://github.com/primer/design/assets/14095146/2d024a23-b7d1-4404-853a-11614b922fc3">

Reply:
Wonderful, looks great! Thank you 🙌